.travis.yml

```diff
@@ -1,24 +1,29 @@
 language: python
 cache: pip
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+      dist: xenial    # required for Python >= 3.7 (travis-ci/travis-ci#9069)
 install:
-  # install dependencies from setup.py
+  # install dependencies from setup.py (there are none for remi)
   - pip install -e .
-  - pip install coverage
+  # install optional packages (for linting, coverage, browser control)
   - pip install flake8
+  - pip install coverage
   # - pip install selenium
+  # install dependencies for specific examples
+  - pip install matplotlib
```

Comment:
There is an advantage to making fewer pip calls as in __pip install coverage flake8 matplotlib  # selenium__ because someday soon pip should have true dependency management across subdependencies (transitive closure).